tests/linux_kernel/vmcore/test_vmcore.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+import unittest
+
+from drgn import Program
+
+VMCORE_PATH = Path("/proc/vmcore")
+
+
+@unittest.skipUnless(VMCORE_PATH.exists(), "not running in kdump")
+class TestAttachToVMCore(unittest.TestCase):
+    def test_attach_to_vmcore(self):
+        prog = Program()
+        prog.set_core_dump("/proc/vmcore")
+        prog.load_debug_info("", default=True, main=True)
```

Comment:
When we add a test case that needs debug info, we should probably have something similar to `LinuxHelperTestCase` (`LinuxVmcoreTestCase`?) that caches the program and handles missing debug info. For this smoke test, I think just remove the `load_debug_info` call; we're just testing whether we can attach to `/proc/vmcore`. Maybe add assertions on `prog.flags` for `!IS_LIVE` and `IS_LINUX_KERNEL` instead.